tools/flang2/flang2exe/ll_structure.c

```diff
@@ -578,6 +578,20 @@ ll_create_module(const char *module_name, const char *target_triple,
 
   compute_ir_feature_vector(new_module, llvm_ir_version);
   compute_datalayout(new_module);
+   
+  #ifdef _WIN32
+  const char *linker_directives[] = {
+     "/DEFAULTLIB:flang.lib",
+     "/DEFAULTLIB:flangrti.lib",
+     "/DEFAULTLIB:ompstub.lib"
+  };
```

Comment:
I don't think we should code them here. There are various options sent to the driver like `-no-flang-libs`, `-static-flang-libs` that affect what should be added here.

Reply:
Should we use a define or what?

Reply:
I was thinking of an option to `flang2.exe`

Reply:
Okay. Another problem is that the LLVM output is not correct. It should be like the debug option, but it's different.